ManatiCR/manatime
# 25: Default tasks set.

**Steps to test**
- Create a project node, and create some tasks nodes to test later.
- Check if the button 'add new task' run successfully.
- Create a time entries node.
- Check if the field project was loaded successfully.
- Check if the 'tasks listing'  section was loaded successfully in the project's node description.

[issue link](https://github.com/ManatiCR/manatime/issues/25)
